src/d3a/d3a_core/sim_results/device_statistics.py

```diff
@@ -55,35 +55,51 @@ def _calc_min_max_from_sim_dict(subdict: Dict, key: str):
 
     @classmethod
     def _device_price_stats(cls, area: Area, subdict: Dict):
-        key_name = "trade_price_eur"
+        sold_key_name = "sold_trade_price_eur"
```

Comment:
Are you certain this is not breaking the simulation results in the UI ? IIRC, this class is also used to generate results for the Load/PV in the UI, therefore it would be great if you could validate that the UI does not fail if you include these changes.